crates/turborepo-ffi/Cargo.toml

```diff
@@ -8,9 +8,17 @@ license = "MPL-2.0"
 crate-type = ["staticlib"]
 
 [dependencies]
+anyhow = "1.0.69"
```

Comment:
FWIW I'm using `thiserror` in #3632

Reply:
I think for FFI thiserror is probably the better choice. turborepo-lib is a little more up in the air.